sunpy/roi/chaincode.py

```diff
@@ -60,44 +60,75 @@ def __init__(self, origin, chaincode, xdelta=1, ydelta=1):
                       y_steps[int(step)] * ydelta]]
 
     def matchend(self, end):
+        """
+        ?
+
+        Parameters
+        ----------
+        end : ?
+
+        Returns
+        -------
+        ?
+
+        """
         return np.alltrue(np.equal(self.coordinates[:, -1], np.asarray(end)))
 
     def matchany(self, coordinates, index):
+        """
+        ?
+
+        Parameters
+        ----------
+        coordinates : ?
+        index : ?
+
+        Returns
+        -------
+        ?
+
+        """
```

Comment:
That was me.  Intended as a placeholder, since in some cases it wasn't
completely clear to me what those functions were meant to do, or what the
parameters were.

I will replace all the ? with "not yet documented" and a todo directive
through a PR to the branch.

On Sat, Jul 25, 2015 at 3:29 PM, ayshih <notifications@github.com> wrote:

> In sunpy/roi/chaincode.py
> <https://github.com/sunpy/sunpy/pull/1497#discussion_r35486111>:
>
> >          return np.alltrue(np.equal(self.coordinates[:, -1], np.asarray(end)))
> >
> >      def matchany(self, coordinates, index):
> > +        """
> > +        ?
> > +
> > +        Parameters
> > +        ----------
> > +        coordinates : ?
> > +        index : ?
> > +
> > +        Returns
> > +        -------
> > +        ?
> > +
> > +        """
>
> Same thing about question marks
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunpy/sunpy/pull/1497/files#r35486111>.
>